tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1210,6 +1211,24 @@ Status Converter::BuildCudaEngine(
         trt_builder_.get(), builder_config.get(), network()));
   }
 
+  std::unique_ptr<nvinfer1::ITimingCache> timing_cache = nullptr;
+  // We only use a timing cache if the algorithm selector is not used.
```

Comment:
Would you please also document the reason behind this decision?

Reply:
I meant, the reason why we don't using timing cache if the algorithm selector is not used?

Reply:
Mostly because TensorRT documentation says not to and they are basically incompatible features. If you don't disable it, you will get many warnings about it, which is a problem currently (we'd like to reduce log verbosity). 

The description is here https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#builder-layer-timing.